.github/workflows/ci.yml

```diff
@@ -33,10 +33,16 @@ jobs:
       # Validate that we can build against the MSRV (minimum specified rust version).
       - run: cargo +1.83.0 build -p foxglove --verbose
       - run: cargo clippy --no-deps --all-targets --tests -- -D warnings
+      - run: cargo +nightly rustdoc -p foxglove-ws-protocol --all-features -- -D warnings --cfg docsrs
       - run: cargo +nightly rustdoc -p foxglove --all-features -- -D warnings --cfg docsrs
       - run: cargo test --features unstable --verbose
         timeout-minutes: 10
+      - run: cargo publish --package foxglove-ws-protocol --dry-run
       - run: cargo publish --package foxglove --dry-run
+      - run: cargo publish --package foxglove-ws-protocol
```

Comment:
Assuming SDK will want to depend on the same version of ws-protocol, we'll need to figure out some step here to wait for this to be published? We can punt on that for now, but something to consider.